packages/gatsby/cache-dir/loader.js

```diff
@@ -388,6 +391,16 @@ export class BaseLoader {
       return false
     }
 
+    // Don't prefetch if this is a crawler bot
+    if (
+      `userAgent` in navigator &&
+      typeof navigator.userAgent !== `undefined`
+    ) {
+      if (BOT_REGEX.test(navigator.userAgent)) {
+        return false
+      }
+    }
```

Comment:
This is a good enough check and saves a few bytes.
```suggestion
    // Don't prefetch if this is a crawler bot
    if (
     navigator.userAgent && BOT_REGEX.test(navigator.userAgent)
    ) {
      return false
    }
```